client/src/store/toolStore.js

```diff
@@ -18,6 +19,36 @@ const getters = {
             return "...";
         }
     },
+    getTools:
+        (state) =>
+        ({ filterSettings }) => {
+            // if no filters
+            if (Object.keys(filterSettings).length == 0) {
+                return [];
+            }
+            const allTools = state.toolsList[0];
```

Comment:
It should not be necessary to have a list with 1 element here. This should just be `state.toolsList`.